notion-core/src/main/scala/com/ambiata/notion/core/LocationIO.scala

```diff
@@ -129,4 +129,48 @@ case class LocationIO(configuration: Configuration, s3Client: AmazonS3Client) {
     case h @ HdfsLocation(path)      => Hdfs.delete(new Path(path)).run(configuration)
   }
 
+  /** get the first lines of a file */
+  def head(location: Location, numberOfLines: Int): RIO[List[String]] = {
```

Comment:
Depending on what conventions we're following would it make more sense to call this `take` and rename `firstLine` to `head`. I'm easy, just a thought.